src/test/java/org/apache/sling/feature/cpconverter/handlers/GroupEntryHandlerTest.java

```diff
@@ -70,4 +70,16 @@ public void parseUser() throws Exception {
         TestUtils.createRepoInitExtension(handler, aclManager, path, getClass().getResourceAsStream(path.substring(1)));
         verify(aclManager, never()).addGroup(any(Group.class));
     }
+
+    @Test
+    public void parseGroupWithConfig() throws Exception {
+        String path = "/jcr_root/system/groups/g/V084LLw1ypl2l9G0e28c/.content.xml";
```

Comment:
my initial reaction was: but that's the same as the hardcoded default..... it's not but with just omitting the 'home' segment it looks similar. i would suggest to use something entirely different like e.g. the default value in apache jackrabbit oak to make it obvious.

i would also suggest to add a test that verifies that groups outside of the configured path are not matched by the parser and thus are ignored.

Reply:
I think the tests are covering that - I did update the path